cpp/tests/bitmask/bitmask_tests.cpp

```diff
@@ -67,14 +68,16 @@ struct CountBitmaskTest : public cudf::test::BaseFixture {
 
 TEST_F(CountBitmaskTest, NullMask)
 {
-  EXPECT_EQ(0, cudf::count_set_bits(nullptr, 0, 32));
+  EXPECT_THROW(cudf::detail::count_set_bits(nullptr, 0, 32, rmm::cuda_stream_default),
+               cudf::logic_error);
+  EXPECT_EQ(32, cudf::detail::valid_count(nullptr, 0, 32, rmm::cuda_stream_default));
 
   std::vector<cudf::size_type> indices = {0, 32, 7, 25};
-  auto counts                          = cudf::segmented_count_set_bits(nullptr, indices);
-  EXPECT_EQ(indices.size(), counts.size() * 2);
-  for (size_t i = 0; i < counts.size(); i++) {
-    EXPECT_EQ(indices[2 * i + 1] - indices[2 * i], counts[i]);
-  }
+  EXPECT_THROW(cudf::detail::segmented_count_set_bits(nullptr, indices, rmm::cuda_stream_default),
+               cudf::logic_error);
+  auto valid_counts =
+    cudf::detail::segmented_valid_count(nullptr, indices, rmm::cuda_stream_default);
```

Comment:
Just adding here for the record that we'd need the equivalent for `segmented_null_count()` and `segmented_count_unset_bits()` as well.
(I know you're on it already.)

Reply:
Those were included in 254a6ccc427c19a5177f5a1d79cb79dc530694b4. Is there something else I missed?

Reply:
Ah, I think I missed this on the first pass.
 
For instance, take [`TEST_F(CountBitmaskTest, MultipleWordsAllBits)`](https://github.com/rapidsai/cudf/commit/254a6ccc427c19a5177f5a1d79cb79dc530694b4#diff-0c93b17acb2ab8ed2e6e0eb179993a325514b415a641dbaa204ecf7d1d9c2e6bR266).
I see that this test calls `count_set`, `valid_count`, `segmented_count_set`, `segmented_valid_count`. 

I didn't realize that the `count_unset`/`null_count` were added to the `CountUnsetBitsTest`. `CountBitmaskTest` is a generic name, so I assumed it would encompass both sets of permutations.

My bad. Please ignore.